Cargo.toml

```diff
@@ -30,6 +30,7 @@ name = "dy"
 path = "src/main.rs"
 
 [dependencies]
+atty            = "0.2"
```

Comment:
This https://crates.io/crates/atty.
MIT licensed.

Reply:
Sure, LGTM :)